src/Seo/SeoStructuredData.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Sideways\Classic\Seo;
+
+use Closure;
+
+class SeoStructuredData
+{
+    private SeoService $seo_service;
+
+    public function __construct() {
+        $this->seo_service = new SeoService();
+    }
+
+    public function run(): void {
+        // Filters for Plugin Metabox on Posts/Pages
+        add_filter( 'the_seo_framework_inpost_seo_save_defaults', static function( $defaults ) {
+            $defaults[SWCLASSIC_POST_SCHEMA_KEY] = '';
+            return $defaults;
+        } );
+
+        add_filter( 'the_seo_framework_inpost_settings_tabs', function( $default_tabs ) {
+            $default_tabs['schema'] = [
+                'name'     => __( 'Schema', '_h' ),
+                'callback' => Closure::fromCallable( [ $this, 'page_schema_metabox' ] ),
+                'dashicon' => 'feedback',
+            ];
+            return $default_tabs;
+        } );
+
+        add_filter( 'the_seo_framework_save_post_meta', static function( $data ) {
+            $data[SWCLASSIC_POST_SCHEMA_KEY] = $_POST['autodescription'][SWCLASSIC_POST_SCHEMA_KEY];
+            return $data;
+        } );
+
+        // Filters for Plugin Settings Page
+        add_filter( 'the_seo_framework_default_site_options', static function( array $settings ) {
+            $settings[SWCLASSIC_SITE_SCHEMA_KEY] = '';
+            return $settings;
+        } );
+
+        add_filter( 'the_seo_framework_schema_settings_tabs', function() {
+            $custom_tabs['custom_structure'] = [
+                'name'     => __( 'Custom Schema Structure', '_h' ),
```

Comment:
change `_h` to something else maybe `swc` or the name of the plugin (this is more standard but more verbose)

Reply:
upated to `sc`